### ISSUE TITLE
Pirsch SDK returns `error: [ 'Access denied.' ]` on stats endpoints

### ISSUE THREAD

**Issue by JustinyAhin** [pirsch-analytics/pirsch-js-sdk#19]
I'm trying to use the JS SDK to get total visitors for my website.

I'm using this endpoint as a reference: https://docs.pirsch.io/api-sdks/api#total-visitors.

Here's how I did it:

```ts
const pirschClient = new Pirsch({
	hostname: 'www.okupter.com',
	clientId: PIRSCH_CLIENT_ID,
	clientSecret: PIRSCH_CLIENT_SECRET,
});

try {
    const stats = await pirschClient.totalVisitors({
        id: PIRSCH_DOMAIN_ID,
        from: new Date(2023, 0, 1),
        to: new Date()
    });

    console.log(stats);
} catch (error) {
    console.error(error);
}
```

But this always returns an error:

<img width="1053" alt="CleanShot 2023-04-09 at 09 31 54@2x" src="https://user-images.githubusercontent.com/33403964/230760237-4ddeba0f-959d-4dbb-8da4-90b150abfe95.png">

Am I doing something wrong?

**Comment by Kugelschieber** [comment]
Hi Justin, thank you for reporting! We'll look into it. I have quickly created a demo on our demo repository and can confirm that there seems to be an issue with the authentication.

https://github.com/pirsch-analytics/demo/tree/master/js-sdk

**Comment by Kugelschieber** [comment]
Fixed in version 2.1.3 :+1:  Thanks again for reporting!

```
npm i pirsch-sdk@2.1.3
```

**Comment by JustinyAhin** [comment]
> Fixed in version 2.1.3 👍 Thanks again for reporting!
> 
> ```
> npm i pirsch-sdk@2.1.3
> ```

Thanks :)
